README.md

```diff
@@ -2,49 +2,50 @@
 
 ![sops-nix logo](https://github.com/Mic92/sops-nix/releases/download/assets/logo.gif "Logo of sops-nix")
 
-Atomic secret provisioning for NixOS based on [sops](https://github.com/mozilla/sops).
+Atomic, declarative, and reproducible secret provisioning for NixOS based on [sops](https://github.com/mozilla/sops).
 
 ## How it works
 
-Sops-nix decrypts secrets [sops files](https://github.com/mozilla/sops#2usage)
-on the target machine to files specified in the NixOS configuration at
-activation time. It also adjusts file permissions/owner/group. It uses either
-age or GPG keys for decryption, where both types can be derived from ssh host
-keys. In future we will also support cloud key management APIs such as AWS
-KMS, GCP KMS, Azure Key Vault or Hashicorp's vault.
+Secrets are decrypted from [`sops` files](https://github.com/mozilla/sops#2usage) during
+activation time. The secrets are access-controlled by by full declarative configuration of their users, permissions, and groups.
+It can use `age`, GPG, or SSH keys for decryption. 
+In the future, we will also support cloud key management APIs such as AWS
+KMS, GCP KMS, Azure Key Vault and Hashicorp Vault.
 
 ## Features
 
-- Compatible with all NixOS deployment frameworks: [NixOps](https://github.com/NixOS/nixops), nixos-rebuild, [krops](https://github.com/krebs/krops/), [morph](https://github.com/DBCDK/morph), [nixus](https://github.com/Infinisil/nixus)
-- Version-control friendly: Since all files are encrypted they can directly committed to version control. The format is readable in diffs and there are also ways of showing [git diffs in cleartext](https://github.com/mozilla/sops#showing-diffs-in-cleartext-in-git)
-- Works well in teams: sops-nix comes with nix-shell hooks that allows quickly import multiple people to import all used keys.
+- Compatible with all NixOS deployment frameworks: [NixOps](https://github.com/NixOS/nixops), nixos-rebuild, [krops](https://github.com/krebs/krops/), [morph](https://github.com/DBCDK/morph), [nixus](https://github.com/Infinisil/nixus), etc.
+- Version-control friendly: Since all files are encrypted they can directly committed to version control without worry. Diffs of the secrets are readable, and [can be shown in cleartext](https://github.com/mozilla/sops#showing-diffs-in-cleartext-in-git).
```

Comment:
```suggestion
- Version-control friendly: Since all files are encrypted they can be directly committed to version control without worry. Diffs of the secrets are readable, and [can be shown in cleartext](https://github.com/mozilla/sops#showing-diffs-in-cleartext-in-git).
```